_drafts/angels_unit_tests.md

```diff
@@ -16,11 +12,11 @@ With the help of the community, we learned how the Angel's libary worked. For th
 
 I am confident now saying that we do not make coding mistakes anymore which would result in a hard crash of the mods. Nowadays the issues are likely edge cases, unhandled cases or a strange modpack configuration. Most frequently we receive bugs from people who play with a few Angel's mods and a few Bob's mods. The reason for this is simple: there are many configurations with many settings, which result in many oversights. [A small example](https://www.reddit.com/r/factorio/comments/pfvqn0/playing_an_angel_bobs_game_and_noticed_about_9/) is shown below: You fix a bug with some technology being unresearchable, in some exotic configuration shown below. The reason why it is unresearchable is simple: It has a prerequisite technology that isn't researched yet, but since the prerequisite is hidden, it does not show up in the technology tree.
 
-![Nickel processing is unresearchable](https://raw.githubusercontent.com/LovelySanta/Alt-F4/angels-unit-tests/submissions/angels_unit_tests/nickel_processing_bug_1.png)
+{% include image.html src='https://media.alt-f4.blog/ALTF4/48/nickel_processing_bug_1.jpg' alt='Nickel processing is unresearchable' %}
```

Comment:
@Conor12345  `Nickel processing is unresearchable` should be the caption too, and not only the alt-text. The same goes for the other images :)